src/OpenSpaceToolkit/Mathematics/Geometry/3D/Objects/Ellipsoid.cpp

```diff
@@ -841,6 +841,11 @@ Intersection                    Ellipsoid::intersectionWith                 (
             const Point firstPoint = EllipsoidPointFromGteVector(intersectionResult.point[0]) ;
             const Point secondPoint = EllipsoidPointFromGteVector(intersectionResult.point[1]) ;
 
+            if (firstPoint == secondPoint)
+            {
+                return Intersection::Point(firstPoint) ;
+            }
+
```

Comment:
Looks like this diff was added to this PR by mistake?

Reply:
Same comment as below.